includes/defaults.inc.php

```diff
@@ -712,6 +712,7 @@
 $config['poller_modules']['ntp']                         = 1;
 $config['poller_modules']['services']                    = 1;
 $config['poller_modules']['loadbalancers']               = 0;
+$config['poller_modules']['mef-evc']                     = 0;
```

Comment:
Well, I dunno. Seems the mef-evc is kinda related to the mib itself. It should be generic and can be used on some other network devices (like adva for example).
You choose what you will like for that... :)